tests/testthat/test_minimize.R

```diff
@@ -185,7 +185,7 @@ test_that("base-case satisfies constraints", {
     )
 
     # compute summaries
-    out <- summary(opt_ts$design, "power" = pow, "toer" = toer)
+    out <- summary(opt_ts$design, "power" = pow, "toer" = toer, rounded = FALSE)
```

Comment:
@MatheMax: i disagree - we want to make sure that the actual design complies, right? So no rounding! are we that much off?